src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/SdkApiVersion.cs

```diff
@@ -7,8 +7,9 @@ public static class SdkApiVersion
         public const int VERSION_2 = 2; // added mouse pixel and object random clicking actions to bot segments
         public const int VERSION_3 = 3; // added ui pixel hash key frame criteria to bot segments
         public const int VERSION_4 = 4; // changed state format to enable entity support including 'type' and 'components' fields; added versioning to 'state' recording not just 'bot_segments'
+        public const int VERSION_5 = 5; // added support for monobehaviour bot segment actions
```

Comment:
you'll want this to be version 6 I think, since 5 adds behaviour support and 6 will add monkey bot support